changes/v4.3.22-en.md

```diff
@@ -27,9 +27,12 @@
 
 - Add `management.bootstrap_apps_file` configuration to bulk import default app/secret when EMQX initializes the database [#9273](https://github.com/emqx/emqx/pull/9273).
 
+- Provide trigger switches for client disconnect events for different reasons [#9267](https://github.com/emqx/emqx/pull/9267).
+  For compatibility with the old behavior, the default configuration is set to the exact same value.
+
```

Comment:
```suggestion
- Added configurations to enable more `client.disconnected` events (and counter bumps) [#9267](https://github.com/emqx/emqx/pull/9267).
  Prior to this change, the disconnected events (and counter bumps) are only triggered when a client
  performs a 'normal' disconnect, or is 'kicked' by system admin.
  However, when a client reconnects without a graceful shutdown, this event is not triggered 
  in case the old connection had to be 'discarded' (for clean session) or 'takenover' (for non-clean session).
  Now it is possible to set configs `broker.client_disconnect_discarded` and `broker.client_disconnect_takenover`
  to `on` to enable the event.

```